amplipi/streams.py

```diff
@@ -253,84 +255,83 @@ def connect(self, src):
     src_config_folder = f'{utils.get_folder("config")}/srcs/{src}'
     toml_template = f'{utils.get_folder("streams")}/spot_config.toml'
     toml_useful = f'{src_config_folder}/config.toml'
+
+    # make source folder
+    os.system(f'mkdir -p {src_config_folder}')
```

Comment:
If this is a problem here, it's going to be a problem for all streams' connect() functions. Should this be done as part of the  `BaseStream` class, or in ctrl.py where it calls `stream.connect(idx)`, or somewhere else that handles this problem for all streams?

Reply:
This was taken care of in #372